src/psyclone/configuration.py

```diff
@@ -237,7 +237,7 @@ def __init__(self):
 
         # By default, the PSyIR backends output argument names on (most)
         # IntrinsicCalls. This option enables control of that behaviour.
-        self._backend_intrinsic_named_kwargs = True
+        self._backend_intrinsic_named_kwargs = False
```

Comment:
Comment above needs changing as the default is now not to put unnecessary argument names?